server/controllers/authController.js

```
@@ -145,12 +149,68 @@ exports.oauthCallback = (req, res) => {
+exports.updateProfileImage = async (req, res) => {
+    const userId = req.userId; // From auth middleware
+
+    try {
+        // Check if file was uploaded
+        if (!req.file) {
+            return res.status(400).json({ message: 'No image file provided' });
+        }
+
+        // Delete old profile image if it exists
+        const existingUser = await User.findByPk(userId);
+        if (existingUser && existingUser.profileImage) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)